model/database/cosmosdb-metrics.yaml

```
@@ -0,0 +1,28 @@
+groups:
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
+    brief: "[Request charge](https://learn.microsoft.com/azure/cosmos-db/request-units) consumed by the operation"
+    instrument: histogram
+    unit: "{request_unit}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.client.active_instance.count
+    type: metric
+    metric_name: db.cosmosdb.client.active_instance.count
```

see https://github.com/open-telemetry/semantic-conventions/issues/1494#issuecomment-2422829241
```suggestion
    metric_name: db.client.cosmosdb.active_instance.count
```

or I think @lmolkova had suggested
```suggestion
    metric_name: db.client.cosmosdb.active_client.count
```

---

she suggested this https://github.com/open-telemetry/semantic-conventions/pull/1438#discussion_r1809142557

---

I have changed this metric name.